chapter-10/end/States/Splash/SplashState.cs

```diff
@@ -10,7 +10,7 @@ public class SplashState : BaseGameState
     {
```

Comment:
Just confirming you meant to modify Chapter-10's end state?

Reply:
yes. The content pipeline was restructured in this chapter so the splashscreen moved to Images/ in the end state.

Reply:
Ok cool - just wanted to be sure 👍